Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bagel.yml

```diff
@@ -47,3 +47,28 @@
           Quantity: 5
         - ReagentId: Nutriment
           Quantity: 5
+
+- type: entity
+  id: FoodBagelCotton
+  parent: FoodBagelBase
+  name: cotton bagel
+  description: A delicious bagel made with cotton dough.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - bread
+      - cotton
+  - type: Sprite
+    state: bagel-cottondough
+  - type: Food
+    requiresSpecialDigestion: true
+  - type: Tag
+    tags:
+      - ClothMade
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 12
+        reagents:
+        - ReagentId: Nutriment
```

Comment:
To have parity with cotton bread, the food solution should be 50% nutriment, 50% cotton of the original amount. They're meant to be primarily for moffs after all!